cmake/DyninstConfig.cmake.in

```diff
@@ -5,6 +5,7 @@ list(INSERT CMAKE_MODULE_PATH 0 "${CMAKE_CURRENT_LIST_DIR}/Modules"
 
 include(DyninstBoost)
 include(DyninstTBB)
+include(DyninstPeParse)
```

Comment:
Like ELF, we want to keep these types of dependencies internal to Dyninst. I wish we didn't export any of our dependencies, but that would be a huge breaking change.

```suggestion

```